checks.d/openvpn.py

```diff
@@ -0,0 +1,42 @@
+import csv
+from collections import defaultdict
+
+# project
+from checks import AgentCheck
+
+
+# Each user really should only have one active connection
+# but empirically, a user can have 3 connections within a 10-second
+# interval without it being the sign of a problem (probably 
+# due to reconnections)
+USER_MAX_CONNECTIONS = 4
+
+class Splunk(AgentCheck):
+    def check(self, instance):
+        filename = instance['path']
+        self.parse_status_file(filename)
+
+
+    def parse_status_file(self, filename):
+        with open(filename) as csvfile:
```

Comment:
What do you think of checking the mtime on the file and emitting a failed service check if it's older than a supplied number of seconds? (Defaulting to not doing the check perhaps?) This would allow us to write a monitor that complained if the file got stale, which would yield useless metrics.